tests/StarWarsValidationTest.php

```diff
@@ -37,7 +38,7 @@ public function testValidatesAComplexButValidQuery() : void
         }
       ';
         $errors = $this->validationErrors($query);
-        self::assertEquals(true, empty($errors));
+        self::assertEquals(true, count($errors) === 0);
```

Comment:
We could replace all these with proper assertions like assertCount

Reply:
SGTM. 6b6e3ad